### PR TITLE
ROS Message Runtime

### PR DESCRIPTION
**Pull request by wolfv**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/message_runtime/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package:
+  name: ros-message-runtime
+  version: 0.4.12
+
+source:
+  url: https://github.com/ros-gbp/message_runtime-release/archive/release/melodic/message_runtime/0.4.12-0.tar.gz
+  fn: ros-message-runtime-0.4.12.tar.gz
+  sha256: 70669611054c91c2d65ae94d5e2474198970929338cbb76411cb560902b1c800
+  patches:
```
Comment (xhochy) [inline recipes/message_runtime/meta.yaml]: Remove empty `patches` section
```suggestion
```

File: recipes/roscpp_core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package:
+  name: ros-roscpp-core
+  version: 0.6.12
+
+source:
+  url: https://github.com/ros-gbp/roscpp_core-release/archive/release/melodic/roscpp_core/0.6.12-0.tar.gz
+  fn: ros-roscpp-core-0.6.12.tar.gz
+  sha256: 814ea5bddeba2cabb4229baa27006fe21dbc56f0fd51bf3afd5e20a22b4582ec
+  patches:
```
Comment (xhochy) [inline recipes/roscpp_core/meta.yaml]: Same
```suggestion
```

File: recipes/roscpp_serialization/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package:
+  name: ros-roscpp-serialization
+  version: 0.6.12
+
+source:
+  url: https://github.com/ros-gbp/roscpp_core-release/archive/release/melodic/roscpp_serialization/0.6.12-0.tar.gz
+  fn: ros-roscpp-serialization-0.6.12.tar.gz
+  sha256: ba4ea435dc1d1680f6242b142f06376785ca3abeed71862c23bd6dd0220833e5
+  patches:
```
Comment (xhochy) [inline recipes/roscpp_serialization/meta.yaml]:
```suggestion
```

**Comment by wolfv** [comment]
@xhochy can we merge as-is? i will fix up my generation script and the next iteration of these packages will not have these issues :) but i am kind of tired of waiting for azure to build

**Pull request merged**